### PR TITLE
Add message broker with queues alongside its respective event consumer

### PR DESCRIPTION
**Pull request by Anthonyxw87**
- [ ] Explain what this PR does
  - added the rabbitmq message broker service. On startup, the meetings, participants, and attachments queues are created on the rabbitmq service. 
  - Then each event consumer service is started that listens to its respective queue. 
  - Included a sample script that describes how to send a message to a specific queue. 
  - Right now, for the event consumer functionality, it just prints out a simple message. Have not implemented the API Gateway and actual processing part yet. 
  - Infrastructure and services are set up though. Now need to incorporate functionality and processing.
- [ ] Documentation updated (if needed)
- [ ] Code reviewed by someone else on the team
- [ ] PR is ready and can be merged (do not check this until all other boxes are checked)

### CODE REVIEW

File: docker-compose.yml
Comment (Anthonyxw87) [inline docker-compose.yml:1]: just deleted the certs and stuff from before. Don't think we need to implement those.

File: event_consumers/attachments_consumer/requirements.txt
Comment (Anthonyxw87) [inline event_consumers/attachments_consumer/requirements.txt:1]: have some dependencies that are not used. Is there a problem leaving it in. Just copied and pasted the requirements.txt file from another service and included the "pika" python library.

File: docker-compose.yml [hunk 1]
```diff
@@ -51,6 +48,36 @@ services:
...
     tty: true
 
+  message_broker:
+    build: ./message_broker
+    ports:
+      - "5672:5672" # Default AMQP RabbitMQ port
+      - "15672:15672" # Management UI port
+    networks:
+      - backend-net
+      - db-net
```
Comment (Jython1415) [inline docker-compose.yml:58]: Why does the message broker have access to the DB network?

File: event_consumers/attachments_consumer/requirements.txt
Comment (Jython1415) [reply event_consumers/attachments_consumer/requirements.txt:1]: If it works, I'd say don't worry about it

**Review by Jython1415 — COMMENTED** [review]
Looks good to me, as far as I can tell. I did have one small question about something in the `docker-compose` though. @Anthonyxw87 can you address that?

File: docker-compose.yml
Comment (Jython1415) [reply docker-compose.yml:1]: Agreed

**Comment by Jython1415** [comment]
Now that we've completed the coding assignments, I'm going to close open PRs without merging them. Of course, you're welcome to to reopen them to get them merged if you feel like finishing the project nonetheless.

**Pull request closed without merging**